projects/aca-content/src/lib/ui/theme.scss

```diff
@@ -49,6 +49,15 @@ mat-toolbar.mat-toolbar.mat-toolbar-single-row {
   }
 }
 
+.mat-checkbox-indeterminate,
+.mat-checkbox-checked {
+  &.mat-accent {
+    .mat-checkbox-background {
+      background-color: var(--theme-blue-checkbox-color);
```

Comment:
Two things: 
1. Did you check if that works in control center? I guess it may work in ACA and ADW because ADW depends on ACA but not sure if that change will fix also checkboxes in control center. 
2. Should not we have it in ADF to keep consistency across all projects which use ADF? Regarding to 2nd point you can discuss it with @MichalKinas to ensure what is best place for it.

Reply:
ok, thank you for verifying these